fe/fe-core/src/main/java/com/starrocks/persist/ModifyPartitionInfo.java

```diff
@@ -137,6 +144,7 @@ public void write(DataOutput out) throws IOException {
 
         out.writeShort(replicationNum);
         out.writeBoolean(isInMemory);
+        out.writeBoolean(dataCacheEnable);
```

Comment:
can't do it like this. double check its compatibility, suppose we should not do any change to write() and readFields(). should only leverage gson serialize to persistent attributes.

Reply:
Discuss it with you face to face tomorrow